.github/config-schema.json

```diff
@@ -1452,6 +1452,7 @@
         ],
         "style": "yellow bold",
         "symbol": "🐍 ",
+        "use_venv_parent_name": false,
```

Comment:
It would be nice if that is clearer if it replaces the logic or is a fallback. So either "prefer_venv_parent_name" or "fallback_to_venv_parent_name" or so...